gitversion.yml

```diff
@@ -28,7 +28,7 @@ branches:
     pre-release-weight: 0
   main:
     mode: ContinuousDeployment
-    tag: ''
+    tag: 'pre'
```

Comment:
I would like to remove that file in the next PR. Versioning can be configured in a more simple way.